### PR TITLE
replace my old vue-draggable-nested-tree with he-tree-vue; add my new vue-colrow

### PR DESCRIPTION
**Pull request by phphe**
### `Type`

> ℹ️  What types of changes does your code introduce?

> 👉 _Put an `x` in the boxes that apply_

- [ ] Fix
- [X] Feature 

### `Checklist`

> ℹ️  Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/CONTRIBUTING.md).

> 👉  _Put an `x` in the boxes that apply._

- [X] Make sure you put things in the right category!
- [X] Always add your items to the end of a list

#### Open Source

- [X] The documentation (README) contains a description of the project, illustration of the project with a demo or screenshots and a CONTRIBUTING section
- [X] The documentation is in English.
- [X] The project is active and maintained.
- [X] The project accepts contributions.

#### Apps/Websites

- [ ] The website is available without errors or ssl certificate problems, and load in a reasonable amount of time.
- [ ] The website is using vuejs intensively. It should detect vue with [vue-devtools](https://github.com/vuejs/vue-devtools).
  > If you cannot detect vue with `vue-devtools` due to work at non public pages (e.g. for enterprise website), you can send Pull Request with screenshot that detected it.
- [ ] The website is original and not too simple. For that reason, blogs and simple landing pages are rejected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1724,7 +1724,7 @@ Tooltips / popovers
  - [vue-drag-it-dude](https://github.com/Esvalirion/vue-drag-it-dude) - Vue2 component, that allows you to drag object wherever you want.
  - [vue-draggable](https://github.com/Vivify-Ideas/vue-draggable) - Vue Drag and Drop library without any dependency. Simple and easy to use.
  - [vue-nestable](https://github.com/rhwilr/vue-nestable) - A simple drag & drop hierarchical list made as a vue component.
- - [vue-draggable-nested-tree](https://github.com/phphe/vue-draggable-nested-tree) - A powerful customizable draggable tree view component for Vuejs2 [@phphe](https://github.com/phphe)
+ - [he-tree-vue](https://he-tree-vue.phphe.com) - A draggable sortable vue tree component, with dragging placeholder, types definition. 14 KB gzipped.
```
Comment (Scrum) [inline README.md]: @phphe double

File: README.md [hunk 2]
```diff
@@ -2161,6 +2162,7 @@ composing CSS breakpoint state.
  - [vue-responsive-components](https://github.com/Kelin2025/vue-responsive-components) - Create responsive components with `ResizeObserver`.
  - [vue-screen-size](https://github.com/promosis/vue-screen-size) - Get easy and reactive access to the width and height of your screen.
  - [vue-provide-responsive](https://github.com/CyberAP/vue-provide-responsive) - Breakpoint-based responsive helpers using Provide\Inject with SSR support
+ - [vue-colrow](https://vue-colrow.phphe.com) - Responsive grid layout components: Row, Col. Based on css flexbox. Support SSR, fixed or fraction width, auto grow Col.
```
Comment (Scrum) [inline README.md]: @phphe double

**Comment by phphe** [comment]
There are 2 repo. One is he-tree-vue, it is a draggable tree, so I put it under 'Tree' and 'Drag and Drop'.
One is vue-colrow, it is a responsive grid layout repo. So I put it under 'UI Layout' and 'Responsive Design'.

File: README.md (hunk 2 above)
Comment (phphe) [reply README.md]: There are 2 repo. One is he-tree-vue, it is a draggable tree, so I put it under 'Tree' and 'Drag and Drop'.
One is vue-colrow, it is a responsive grid layout repo. So I put it under 'UI Layout' and 'Responsive Design'.

**Comment by Scrum** [comment]
@phphe 

> There are 2 repo. One is he-tree-vue, it is a draggable tree, so I put it under 'Tree' and 'Drag and Drop'.

I think `draggable` is a tree option and it’s more logical to leave it only in the tree category

> One is vue-colrow, it is a responsive grid layout repo. So I put it under 'UI Layout' and 'Responsive Design'.

The grid is more suitable for `UI` category and its option is `Responsive`

**Comment by phphe** [comment]
> @phphe Could you update this PR if it is relevant. In any case, let me know if something is necessary.
> 
> ## Checklist
> * [x]  git: link not to repository
> * [ ]  git: broken PR
> * [ ]  git: no conflicts resolved
> * [ ]  time: not updated for more than 6 months
> * [ ]  category: not in its category
> * [ ]  lists: not the last on the list
> * [ ]  docs: no documentation
> * [ ]  docs: non-english documentation
> * [ ]  project: dead project
> * [ ]  project: does not accept donat
> * [ ]  site: no link to the site
> * [ ]  site: site with errors
> * [ ]  site: not actively using vuejs
> * [ ]  site: not original
> 
> > more about [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/CONTRIBUTING.md)

The link is to docs site.

**Comment by Scrum** [comment]
@phphe 
> The link provided should be the link of the github repository, not the demo. If you consider your project finished, and it does not need any contributions, consider the apps/website category.

read please [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/.github/contributing.md#open-source)

**Comment by phphe** [comment]
> @phphe Could you update this PR if it is relevant. In any case, let me know if something is necessary.
> 
> ## Checklist
> * [ ]  git: link not to repository
> * [ ]  git: broken PR
> * [ ]  git: no conflicts resolved
> * [ ]  time: not updated for more than 6 months
> * [ ]  category: not in its category
> * [ ]  lists: not the last on the list
> * [ ]  docs: no documentation
> * [ ]  docs: non-english documentation
> * [ ]  project: dead project
> * [ ]  project: does not accept donat
> * [ ]  site: no link to the site
> * [ ]  site: site with errors
> * [ ]  site: not actively using vuejs
> * [ ]  site: not original
> 
> > more about [CONTRIBUTING](https://github.com/vuejs/awesome-vue/blob/master/CONTRIBUTING.md)

Now they are linked to repo.

**Comment by Scrum** [comment]
@phphe uhh, like that, thanks a lot

**Comment by phphe** [comment]
thank you

**Pull request merged**